spec/generators/install_generator_spec.rb

```diff
@@ -1,27 +1,33 @@
-# -*- encoding: utf-8 -*-
```

Comment:
I remembered that this was no longer needed, but not why. It's not needed for ruby 2.0, so as let's remove all of these when we drop support for 1.9